core/lib/page-functions.js

```diff
@@ -132,10 +142,9 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
       }
 
       // Elide attribute value if too long.
-      if (attributeValue.length > ATTRIBUTE_CHAR_LIMIT) {
-        attributeValue = attributeValue.slice(0, ATTRIBUTE_CHAR_LIMIT - 1) + '…';
-        dirty = true;
-      }
+      const truncatedString = truncate(attributeValue, ATTRIBUTE_CHAR_LIMIT);
+      dirty ||= truncatedString !== attributeValue;
```

Comment:
nit: `||=` is hard to understand here IMO

```suggestion
      if(!dirty) dirty = (truncatedString !== attributeValue);
```

Reply:
I have the opposite opinion here 🤷